bin/solr/update_document

```diff
@@ -37,13 +37,15 @@ def update_document_type_form(db_id, solr_id)
   begin
     solr_form_doc = JSON.load(db_form[:data]).
       merge(JSON.load(db_form[:processed_data])).
-      merge({
-        id: db_form[:id],
-        kind_s: db_form[:kind],
-        review_s: db_form[:review],
-        user_i: db_form[:user_id],
-        parent_db_form_i: db_form[:parent_id]
-      })
+      merge(
+        {
+          id: db_form[:id],
+          kind_s: db_form[:kind],
+          review_s: db_form[:review],
+          user_i: db_form[:user_id],
+          parent_db_form_i: db_form[:parent_id]
+        }
+      )
```

Comment:
I wonder if there's a way to override the [rule](http://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/MultilineMethodCallBraceLayout) for nested brackets? I prefer the former syntax for these situations, and don't think it should be disallowed.

It appears that the default `symmetrical` option would allow that. However it would also allow stuff we don't want like 
```
method(a,
  b)
```
:(